<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="38" passed="30" failed="8" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-06-15T09:34:23 IST" name="softwaretestingmaterial" finished-at="2020-06-15T09:34:43 IST" duration-ms="20558">
    <groups>
    </groups>
    <test started-at="2020-06-15T09:34:23 IST" name="testngTest" finished-at="2020-06-15T09:34:43 IST" duration-ms="20558">
      <class name="com.restapi_TestCases.TC002_Get_Single_EmployeedDetails">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:25 IST" name="SetUp" finished-at="2020-06-15T09:34:25 IST" duration-ms="27" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method is-config="true" signature="getEmplpoyeeData()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:25 IST" name="getEmplpoyeeData" finished-at="2020-06-15T09:34:26 IST" duration-ms="992" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmplpoyeeData -->
        <test-method signature="checkContentLength()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:26 IST" name="checkContentLength" finished-at="2020-06-15T09:34:26 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentLength -->
        <test-method signature="checkCookies()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:26 IST" name="checkCookies" finished-at="2020-06-15T09:34:26 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCookies -->
        <test-method signature="checkReponseTime()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:26 IST" name="checkReponseTime" finished-at="2020-06-15T09:34:26 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkReponseTime -->
        <test-method signature="checkResponseBody()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:26 IST" name="checkResponseBody" finished-at="2020-06-15T09:34:26 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.restapi_TestCases.TC002_Get_Single_EmployeedDetails.checkResponseBody(TC002_Get_Single_EmployeedDetails.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseBody -->
        <test-method signature="checkServerType()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:27 IST" name="checkServerType" finished-at="2020-06-15T09:34:27 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkServerType -->
        <test-method signature="checkStatusCode()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:27 IST" name="checkStatusCode" finished-at="2020-06-15T09:34:27 IST" duration-ms="2" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.restapi_TestCases.TC002_Get_Single_EmployeedDetails.checkStatusCode(TC002_Get_Single_EmployeedDetails.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method signature="checkStatusLine()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:27 IST" name="checkStatusLine" finished-at="2020-06-15T09:34:27 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.restapi_TestCases.TC002_Get_Single_EmployeedDetails.checkStatusLine(TC002_Get_Single_EmployeedDetails.java:78)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusLine -->
        <test-method signature="checkTheContentType()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:27 IST" name="checkTheContentType" finished-at="2020-06-15T09:34:27 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTheContentType -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.restapi_TestCases.TC002_Get_Single_EmployeedDetails@1ffe63b9]" started-at="2020-06-15T09:34:27 IST" name="tearDown" finished-at="2020-06-15T09:34:27 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.restapi_TestCases.TC002_Get_Single_EmployeedDetails -->
      <class name="com.restapi_TestCases.TC001_Get_All_Employees">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:23 IST" name="SetUp" finished-at="2020-06-15T09:34:23 IST" duration-ms="86" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method is-config="true" signature="getAllEmployeeDetails()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:23 IST" name="getAllEmployeeDetails" finished-at="2020-06-15T09:34:25 IST" duration-ms="2569" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllEmployeeDetails -->
        <test-method signature="checkContentEncodeType()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkContentEncodeType" finished-at="2020-06-15T09:34:25 IST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentEncodeType -->
        <test-method signature="checkContentLength()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkContentLength" finished-at="2020-06-15T09:34:25 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentLength -->
        <test-method signature="checkCookies()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkCookies" finished-at="2020-06-15T09:34:25 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCookies -->
        <test-method signature="checkReponseTime()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkReponseTime" finished-at="2020-06-15T09:34:25 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkReponseTime -->
        <test-method signature="checkResponseBody()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkResponseBody" finished-at="2020-06-15T09:34:25 IST" duration-ms="20" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseBody -->
        <test-method signature="checkServerType()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkServerType" finished-at="2020-06-15T09:34:25 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkServerType -->
        <test-method signature="checkStatusCode()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkStatusCode" finished-at="2020-06-15T09:34:25 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method signature="checkStatusLine()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkStatusLine" finished-at="2020-06-15T09:34:25 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusLine -->
        <test-method signature="checkTheContentType()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="checkTheContentType" finished-at="2020-06-15T09:34:25 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTheContentType -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.restapi_TestCases.TC001_Get_All_Employees@5dd6264]" started-at="2020-06-15T09:34:25 IST" name="tearDown" finished-at="2020-06-15T09:34:25 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.restapi_TestCases.TC001_Get_All_Employees -->
      <class name="com.restapi_TestCases.TC005_Delete_EmployeeRecord">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:41 IST" name="SetUp" finished-at="2020-06-15T09:34:41 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method is-config="true" signature="getAllEmployeeDetails()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:41 IST" name="getAllEmployeeDetails" finished-at="2020-06-15T09:34:43 IST" duration-ms="2278" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllEmployeeDetails -->
        <test-method signature="checkContentEncodeType()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkContentEncodeType" finished-at="2020-06-15T09:34:43 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentEncodeType -->
        <test-method signature="checkContentLength()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkContentLength" finished-at="2020-06-15T09:34:43 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentLength -->
        <test-method signature="checkCookies()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkCookies" finished-at="2020-06-15T09:34:43 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCookies -->
        <test-method signature="checkReponseTime()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkReponseTime" finished-at="2020-06-15T09:34:43 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkReponseTime -->
        <test-method signature="checkResponseBody()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkResponseBody" finished-at="2020-06-15T09:34:43 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.restapi_TestCases.TC005_Delete_EmployeeRecord.checkResponseBody(TC005_Delete_EmployeeRecord.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseBody -->
        <test-method signature="checkServerType()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkServerType" finished-at="2020-06-15T09:34:43 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkServerType -->
        <test-method signature="checkStatusCode()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkStatusCode" finished-at="2020-06-15T09:34:43 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method signature="checkStatusLine()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkStatusLine" finished-at="2020-06-15T09:34:43 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusLine -->
        <test-method signature="checkTheContentType()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="checkTheContentType" finished-at="2020-06-15T09:34:43 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTheContentType -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.restapi_TestCases.TC005_Delete_EmployeeRecord@12e61fe6]" started-at="2020-06-15T09:34:43 IST" name="tearDown" finished-at="2020-06-15T09:34:43 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.restapi_TestCases.TC005_Delete_EmployeeRecord -->
      <class name="com.restapi_TestCases.TC004_update_Existing_EmployeeRecord">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:31 IST" name="SetUp" finished-at="2020-06-15T09:34:31 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method is-config="true" signature="postNewEmployee()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:31 IST" name="postNewEmployee" finished-at="2020-06-15T09:34:41 IST" duration-ms="10061" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postNewEmployee -->
        <test-method signature="checkContentEncodingType()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:41 IST" name="checkContentEncodingType" finished-at="2020-06-15T09:34:41 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentEncodingType -->
        <test-method signature="checkContentType()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:41 IST" name="checkContentType" finished-at="2020-06-15T09:34:41 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [application/json;charset=utf-8] but found [text/html;charset=UTF-8]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [application/json;charset=utf-8] but found [text/html;charset=UTF-8]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.restapi_TestCases.TC004_update_Existing_EmployeeRecord.checkContentType(TC004_update_Existing_EmployeeRecord.java:80)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentType -->
        <test-method signature="checkServerType()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:41 IST" name="checkServerType" finished-at="2020-06-15T09:34:41 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkServerType -->
        <test-method signature="checkStatusCode()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:41 IST" name="checkStatusCode" finished-at="2020-06-15T09:34:41 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [405]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.restapi_TestCases.TC004_update_Existing_EmployeeRecord.checkStatusCode(TC004_update_Existing_EmployeeRecord.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method signature="checkStatusLine()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:41 IST" name="checkStatusLine" finished-at="2020-06-15T09:34:41 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusLine -->
        <test-method signature="postRequestValidation()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:41 IST" name="postRequestValidation" finished-at="2020-06-15T09:34:41 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.restapi_TestCases.TC004_update_Existing_EmployeeRecord.postRequestValidation(TC004_update_Existing_EmployeeRecord.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postRequestValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.restapi_TestCases.TC004_update_Existing_EmployeeRecord@7c469c48]" started-at="2020-06-15T09:34:41 IST" name="tearDown" finished-at="2020-06-15T09:34:41 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.restapi_TestCases.TC004_update_Existing_EmployeeRecord -->
      <class name="com.restapi_TestCases.TC003_Post_New_Employee">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:27 IST" name="SetUp" finished-at="2020-06-15T09:34:27 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method is-config="true" signature="postNewEmployee()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:27 IST" name="postNewEmployee" finished-at="2020-06-15T09:34:31 IST" duration-ms="4301" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postNewEmployee -->
        <test-method signature="checkContentEncodingType()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:31 IST" name="checkContentEncodingType" finished-at="2020-06-15T09:34:31 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [gzip] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEqualsImpl(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.restapi_TestCases.TC003_Post_New_Employee.checkContentEncodingType(TC003_Post_New_Employee.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentEncodingType -->
        <test-method signature="checkContentType()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:31 IST" name="checkContentType" finished-at="2020-06-15T09:34:31 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentType -->
        <test-method signature="checkServerType()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:31 IST" name="checkServerType" finished-at="2020-06-15T09:34:31 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkServerType -->
        <test-method signature="checkStatusCode()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:31 IST" name="checkStatusCode" finished-at="2020-06-15T09:34:31 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method signature="checkStatusLine()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:31 IST" name="checkStatusLine" finished-at="2020-06-15T09:34:31 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusLine -->
        <test-method signature="postRequestValidation()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:31 IST" name="postRequestValidation" finished-at="2020-06-15T09:34:31 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postRequestValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.restapi_TestCases.TC003_Post_New_Employee@51e5fc98]" started-at="2020-06-15T09:34:31 IST" name="tearDown" finished-at="2020-06-15T09:34:31 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.restapi_TestCases.TC003_Post_New_Employee -->
    </test> <!-- testngTest -->
  </suite> <!-- softwaretestingmaterial -->
</testng-results>
